tool/lib/commands/build.dart

```diff
@@ -110,6 +110,7 @@ class BuildCommand extends Command {
             if (buildMode == 'debug') '--dart2js-optimization=O1',
             if (buildMode != 'debug') '--$buildMode',
           ],
+          '-O2',
```

Comment:
Dart2js is very slow in `-O2` mode (I've been told). You can mark it using -O4 while wasm use -O2 via
```
// Use -O2 optimization (sound dart) but override for dart2js with -O4 (it's too slow otherwise)
if (buildMode != 'debug') ...['-O2', '--dart2js-optimization=O4']
```